src/openMVG/features/descriptor.hpp

```diff
@@ -68,14 +68,19 @@ class Descriptor
       res[i] = data[i] / other[i];
     return res;
   }
-  
+
   inline This& operator*=(const value_type scalar) 
   {
     for(size_type i = 0; i < size(); ++i)
       data[i] *= scalar;
     return *this;
   }
 
+  bool operator==(const Descriptor& other) const
+  {
+    return std::equal(data, data+N, other.data);
```

Comment:
ok for integer values, we should remember it may not work well for float/double in case...